src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -24,6 +24,7 @@ namespace System.Net.Http.Functional.Tests
     public class HttpClientHandlerTest
     {
         readonly ITestOutputHelper _output;
+        private const string ExpectedContent = "Test contest";
```

Comment:
>Nit: I think you meant "Test content"

Thx. We all missed that. I will fix it later in the week when I do my next set of test changes.